ui/app/styles/core/breadcrumb.scss

```diff
@@ -11,7 +11,24 @@
     }
   }
 
-  li.is-active a {
+  ul {
+    align-items: center;
+  }
+
+  li::before {
+    font-size: x-large;
+  }
+
+  li:last-child a.active {
```

Comment:
The `LinkTo` component is connected to the route layer and gets an active property if the route is active.

We use `last-child` if we're in a nested route because the parent route will also be set to active.

There's a namespace related issue where if the `namespace` query parameter is not forwarded then `LinkTo` will not be set to active. This is the same root cause as noted in [11327](https://github.com/hashicorp/nomad/issues/11327) and is noted by the [Ember Core team](https://github.com/emberjs/ember.js/issues/18683).

We're watching the issue and all links will be properly highlighted when the issue is resolved by Ember Core and we don't think its worthwhile to create a workaround solution as this will be fixed by Core soon.

![image](https://user-images.githubusercontent.com/41024828/146577726-68759792-ba95-4ba9-bbf1-595be68739d1.png)